cf2/Fields/RenderField.php

```diff
@@ -130,6 +130,7 @@ public function data()
                 'multiUploadText' => ! empty($this->field[ 'config' ]['multi_upload_text' ]) ? $this->field[ 'config' ]['multi_upload_text' ] : false,
                 'allowedTypes' => $this->getAllowedTypes(),
                 'control' => uniqid($this->getOuterIdAttr() ),
+                'usePreviews' => ! empty($this->field[ 'config']['use_image_previews']) ? $this->field[ 'config']['use_image_previews'] : false,
```

Comment:
@New0 Small note - only valid values we want to pass to DOM are true and false, but what if `$this->field[ 'config']['use_image_previews']` is `pants` or `['foo' => 'bar' ]` ?

`! empty($this->field[ 'config']['use_image_previews']) ? true : false` removes a whole class of edge cases.